### DISCUSSION TITLE
各位大佬，游戏联机都是IP地址，怎么才能规避流量不走代理？

### DISCUSSION THREAD

**Discussion by hxy310** [vernesong/OpenClash#3822] [Q&A]
![photo_2024-03-22_22-36-28](https://github.com/vernesong/OpenClash/assets/42276845/9a890153-0832-42ec-8405-6e854aa9ccc0)

**Comment by VMatrices** [comment]
fake-ip

**Reply by hxy310** [reply]
切换fake-ip后还需要设置规则吗，如果需要该怎么设置，能指点一下吗

**Reply by smyhw** [reply]
fakeip下走dns的东西都被解析到198.18.0.0段了，那这时候可以加一条规则`iptables -t mangle -I openclash 1 ! -d 192.168.0.0/16 -j RETURN`让所有ip直连的流量绕过tun接口

* 仅供参考

**Comment by xiaoyangdkj** [comment]
游戏都走的udp通道，udp转发关了就行了
